packages/console/src/components/CreateTenantModal/SelectTenantPlanModal/PlanCardItem/index.tsx

```diff
@@ -121,11 +92,9 @@ function PlanCardItem({ plan, onSelect }: Props) {
           onClick={onSelect}
         />
       </div>
-      {planId === ReservedPlanId.Pro ||
-        // Todo @xiaoyijun [Pricing] Remove feature flag
-        (isDevFeaturesEnabled && planId === ReservedPlanId.Hobby && (
-          <div className={styles.mostPopularTag}>{t('most_popular')}</div>
-        ))}
+      {planId === ReservedPlanId.Hobby && (
```

Comment:
Pro?

Reply:
The new 'pro' plan id is 'hobby' now, will derepcated after launch. keep for compatibility.